src/test/lib/render_test.ts

```diff
@@ -1409,4 +1409,43 @@ suite('render()', () => {
       assert(container.innerHTML, '<div></div>');
     });
   });
+
+  // `render` directly passes the given value to `NodePart#setValue`, so these
+  // tests are really just a sanity check that they are accepted by `render`.
+  // Tests about rendering behavior for specific values should generally be
+  // grouped with those of `NodePart#setValue` and `#commit`.
+  suite('accepts types other than TemplateResult', () => {
```

Comment:
Let's add a test for accepting a Node since that seems like it might not be uncommon.

Reply:
Added at the bottom.